packages/eas-cli/src/submissions/ios/utils/language.ts

```diff
@@ -0,0 +1,49 @@
+import log from '../../../log';
+
+export const LANGUAGES = [
+  'Brazilian Portuguese',
+  'Danish',
+  'Dutch',
+  'English',
+  'English_Australian',
+  'English_CA',
+  'English_UK',
+  'Finnish',
+  'French',
+  'French_CA',
+  'German',
+  'Greek',
+  'Indonesian',
+  'Italian',
+  'Japanese',
+  'Korean',
+  'Malay',
+  'Norwegian',
+  'Portuguese',
+  'Russian',
+  'Simplified Chinese',
+  'Spanish',
+  'Spanish_MX',
+  'Swedish',
+  'Thai',
+  'Traditional Chinese',
+  'Turkish',
+  'Vietnamese',
+];
+
+/**
+ * Validates language for `travelingFastlane app_produce` command.
+ * @param lang Language to validate
+ * @returns Provided language if valid
+ * @throws Error if language is invalid.s
+ */
+export function validateLanguage(lang?: string): string | undefined | never {
```

Comment:
This is an opinion but:
- when a function name is `validateXYZ`, it should return a boolean
- when you want to have a function which both validates sth and returns the standardized output, it should be called `sanitizeXYZ`

Reply:
Also, why `never`?

Reply:
> Also, why `never`?

Because this function can result in throwing exception

Reply:
It's not required, see https://www.typescriptlang.org/play?#code/MYewdgzgLgBAMgQQHIHECqCUFEDKMC8MA2gFAwwDkAQgE4CGAXgJYA2TdYMACiDVAK4BzfgFMIIigBoylACIcmEABZSZFWfyjAV08hSxhBbZar0GjipQH0E-aPTYdTlc8esBhBM-2G3VtADS3gBiTGBgliE0ImDaUTHaVp7eKCI0ALZOupQo0SIA1t4AkmAAJuBi7GDFUHSO1dkUAFJ0AA4cYhKNAbwiWWoAsnV0AJ7eSLwA7iKCVd48fEKi4t4ASnYQc404TOmtbABmTCKlMO5KYZ3eOO0RJtu3llYDABrX06WRjQAqSnRM3m+9E+UCY4DqZwuYCuP34NHyXzUADVjlAwHR0jCSABdADcJBIAHoAFTEsjEmBIupMUp0KBiGAsDjCOiCEQwA68GAAAyg9AAbiI2IZgnRoEzoTA2q0rK0aCBSvxgCJuTBQOlMmUAHTkmAAAXa9HSjOZ8GZ-FZ7KgIBg-OptPpur10QENEg3Hl-JpJxNhgtbJgTAOtvtTqgSnlkwgMCwNHlNEDwYlLIDikDYDtbFKWog5MJJAO-FioPAIazdJEcHNloAFMmAPwALhg9jCggAlM3W4YYAAfGBF0oiI7Q04AbxkQZgddNADJZzAAISIVAYbA4LVhYAsfhDiAzwzt9swCfkcigSCwZNwRSwQgr9CYXBazKtA+CAgAPh5AFoYAASMdkwAX25dstQAKxAMIawoAAdap23xGRyHDSMYGhSYYzjXga25ABNEB+BgdI7FgCBWhEYAgxGKUW34VpWl4elTmTf0RC1GAcAYpi+B9NjLWjOhokbBDAOvW9QKQmRgIJcgXThThk3xWSSD5WjT0zGkKyrP1awoHgWAgBEKGk4C1TpbRpzSGhj1PNVwAgEAWA4mzcJssyCRIC8nJcrUWBAQQay0h1K2rNlYNcSIj1xIA

Reply:
Of course it's not, I just wanted to be explicit ;)